src/main/java/Converter.java

```diff
@@ -0,0 +1,24 @@
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Converter {
+    public List<Integer> convert(List<String> numberCandidates) {
+        try {
+            List<Integer> numbers = numberCandidates.stream().map(Integer::parseInt).collect(Collectors.toList());
```

Comment:
개행을 하게 되면 조금 더 가독성이 높아질 것 같은데 어떻게 생각하시나요?
```suggestion
            List<Integer> numbers = numberCandidates.stream()
                                    .map(Integer::parseInt)
                                    .collect(Collectors.toList());
```

Reply:
개행하는것이 훨씬 가독성이 좋아보입니다!